app/assets/javascripts/admin/addon/routes/admin-emojis-settings.js

```diff
@@ -0,0 +1,20 @@
+import DiscourseRoute from "discourse/routes/discourse";
+import I18n from "discourse-i18n";
+import SiteSetting from "admin/models/site-setting";
+
+export default class AdminEmojisSettingsRoute extends DiscourseRoute {
+  queryParams = {
+    filter: { replace: true },
+  };
+
+  titleToken() {
+    return I18n.t("settings");
+  }
+
+  async model() {
+    return {
+      settings: await SiteSetting.findAll(),
+      initialFilter: "emoji",
```

Comment:
Emoji don't have their own category, so using initial filter here.

Reply:
Adding an emoji `area` to all the site settings is the way to go here, then you can use `filter_area: "emoji"` when fetching the settings from the server using `SiteSetting.findAll`